budgetportal/views.py

```diff
@@ -821,8 +821,7 @@ def resources(request):
 
 
 def guides(request, slug):
-    if slug not in guide_data:
-        return HttpResponse(status=404)
+    slug = "index"
```

Comment:
this doesn't look like a good thing to merge at the moment

Reply:
true